app/Models/UserLedgerEntry.php

```diff
@@ -81,6 +81,12 @@ class UserLedgerEntry extends Model
         'amount' => 'int',
     ];
 
+    public static function waitingPayments(): int
+    {
+        return (int) self::whereIn('status', [self::STATUS_PROCESSING, self::STATUS_PENDING, self::STATUS_BLOCKED])
```

Comment:
```suggestion
        return (int)self::whereIn('status', [self::STATUS_PROCESSING, self::STATUS_PENDING, self::STATUS_BLOCKED])
```

Reply:
Thanks! It changes everything! Now the code is so beautiful ;-)